crates/ty_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -81,7 +133,7 @@ pub(crate) fn path_to_module(db: &dyn Db, path: &FilePath) -> Option<Module> {
 ///
 /// Returns `None` if the file is not a module locatable via any of the known search paths.
 #[salsa::tracked(heap_size=get_size2::GetSize::get_heap_size)]
-pub(crate) fn file_to_module(db: &dyn Db, file: File) -> Option<Module> {
+pub fn file_to_module(db: &dyn Db, file: File) -> Option<Module> {
```

Comment:
this looks like it's still only used inside this crate, and I'm kinda leery about exposing it outside this crate

```suggestion
pub(crate) fn file_to_module(db: &dyn Db, file: File) -> Option<Module> {
```